packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -0,0 +1,73 @@
+import {
+    ApiQueryResults,
+    Explore,
+    findFieldByIdInExplore,
+    friendlyName,
+    getFieldLabel,
+} from 'common';
+import { useEffect, useState } from 'react';
+
+const useBigNumberConfig = (
```

Comment:
I like the idea of moving it all in here!

makes sense that this will accept the `bigNumberConfig` from the viz provider?